crates/pixi_manifest/src/manifests/table_name.rs

```diff
@@ -65,29 +66,34 @@ impl TableName<'_> {
         let mut parts = Vec::new();
 
         if self.prefix.is_some() {
-            parts.push(self.prefix.unwrap());
+            parts.push(self.prefix.unwrap().to_string());
         }
 
         if self
```

Comment:
While you are at it, you can also use if let some here to get rid of the expect inside the if block.